packages/bitcore-wallet-service/src/lib/chain/eth/index.ts

```diff
@@ -467,9 +467,16 @@ export class EthChain implements IChain {
       address = Web3.utils.toChecksumAddress(tx.abiType.params[0].value);
       amount = tx.abiType.params[1].value;
     } else if (tx.abiType && tx.abiType.type === 'MULTISIG' && tx.abiType.name === 'confirmTransaction') {
+      let amount = 0,
+        address = '0x0';
       multisigContractAddress = tx.to;
-      address = Web3.utils.toChecksumAddress(tx.internal[0].action.to);
-      amount = tx.internal[0].action.value;
+      if (tx.internal) {
```

Comment:
I believe tx.internal is a pre-set empty array, so this is truthy. Should be `if (tx.internal && tx.internal.length > 0)`